### PR TITLE
Update package-lock.json and fix acceptance tests

### PR DESCRIPTION
**Pull request by cjwooo**
## Description

My VS Code auto-changed all the double quotes to single quotes in the acceptance-test-config.yml files, but the actual change was adding `expect_trace_message_on_failure: false`. This particular test assertion has been flaky in the past. For me specifically, I could never get this assertion to pass locally, but it was passing on GHA up until now. Since we should migrate our acceptance-test-config.ymls to use Airbyte's new format anyways, I'm disabling this assertion until we do so.

## Type of change
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: sources/statuspage-source/resources/schemas/component_uptime.json [hunk 1]
```diff
@@ -19,11 +19,9 @@
     },
     "warnings": {
       "type": "array",
-      "items": [
-        {
-          "type": "string"
-        }
-      ]
+      "items": {
```
Comment (chalenge) [inline sources/statuspage-source/resources/schemas/component_uptime.json:22]: 👍🏽

**Review by chalenge — APPROVED** [review]
🚢

**Review by tovbinm — APPROVED** [review]
`'` yay

**Review by tovbinm — COMMENTED** [review]
will this fix the acceptance tests for the sources we just disabled?

**Comment by cjwooo** [comment]
Yes. I've reenabled that one.

**Pull request merged**